### ISSUE TITLE
Take advantage of more acousticsim options

### ISSUE THREAD

**Issue by mmcauliffe** [kchall/CorpusTools#305]
Other representations in python-acoustic-similarity:
AC pitch (Praat and python implementations)
LPC formants (Praat, FAVE, and python implementations)
Gammatone filter banks
Mean Hilbert Envelope Coefficients (MHEC, combines MFCCs and gammatones filter banks)

Other distance metrics:
Average euclidean distance of discrete cosine transform coefficients (DCT distance)
Point measures (Midpoint and third through the vowel)

Other group comparisons:
Performance of clustering algorithms
Distance between perfect clusters (use true labels to create clusters)
Average pairwise distance (current)

Clustering algorithms (sklearn.cluster):
Affinity propagation (many clusters, uneven size, poor scalability)
Spectral clustering (few clusters, even size, medium scalability)
Agglomerative clustering (Many clusters, good scalability)
DBSCAN (uneven cluster sizes, good scalability)
